File: papeterie/ariel/code.js
const axios = require('axios');

axios
  get('https://ariel.pfregistres.com/apiUnlocked/all/{siren}')
  .then(res => {
    console.log(`statusCode: ${res.status}`);
    console.log(res);
  })
  .catch(error => {
    console.error(error);
  });

File: papeterie/ariel/code.java
URL url = new URL("https://ariel.pfregistres.com/apiUnlocked/all/{siren}");

HttpURLConnection connection = (HttpURLConnection) url.openConnection();

connection.setRequestProperty("accept", "application/json");

InputStream responseStream = connection.getInputStream();

ObjectMapper mapper = new ObjectMapper();
APOD apod = mapper.readValue(responseStream, APOD.class);

System.out.println(apod.title);